maths/average_welford.py

```diff
@@ -0,0 +1,21 @@
+def average_welford(values: list) -> float:
```

Comment:
thank you! i created new branches and it seemed to work. is it possible that i might've made a mistake while comitting?? i'm not sure how to check the result of the commit so i might've not seen that they commited as a group.

Reply:
So @meg-1, what you may try is to create 1 branch for 1 file (with its related changes if it applies). You may:
1. checkout to the master branch
2. switch to a new branch, say `average-welford`
3. `git add` and `git commit` only `maths/average_welford.py`
4. check using `git status`
5. push it to your repository; and open a PR for that branch
6. repeat 1-5 for other files

If you have one commit for each file, it's very easy to do this with `cherry-pick` otherwise you'll need some `rebase` which is a bit complicated.

Reply:
thank you, @poyea, for the instruction! that's what i did before in my previous prs ( #6052 #6061 #6077 ). are they being displayed as one joined commit, with 3 files in it? if so then im not sure what could have gone wrong when i commited them :')

Reply:
Yes. They are (you can see it in `Files changed` tab). If you follow the push workflow above, you get one file in each branch, linked to one PR @meg-1

Reply:
> Yes. They are (you can see it in `Files changed` tab). If you follow the push workflow above, you get one file in each branch, linked to one PR @meg-1

ty for clarification. just checked and somehow they really did merge into one commit, 'is narcissistic' [ this pr, #6052 ]. i remember you said that the CI can sometimes not work properly, and i think this is the case for me. i did all prs exactly by the instruction steps which you recommended above, and it still merged into one pr. would you please suggest a further course of action, since i've been trying to commit for quite a long time, and even though i've been following all steps exactly, nothing seems to go through